gensim/test/test_nmf.py

```diff
@@ -87,18 +87,18 @@ def testTransform(self):
         transformed = self.model[doc]
 
         vec = matutils.sparse2full(transformed, 2)  # convert to dense vector, for easier equality tests
-        expected = [0., 1.]
+        expected = [0.02991635, 0.97008365]
```

Comment:
Why have these expected values changed? Could you please explain briefly?

Reply:
Sure. This PR is WIP for now because some AppVeyor (Windows) tests fail though they work on my local Linux machine. Travis doesn't run at all for some reason.

The `testTransform` test returns a slightly different vector on Windows machine than it does on the Linux machine, I'm not sure why. Seems like it's not related to the random state.

Reply:
My preference would be to choose a bigger corpus, one where the model converges reliably and we don't have to rely on RNG so much. Should RNG seed differences really result in vastly different models? Is NMF so unstable?